src/lib/Scenes/Favorites/index.tsx

```diff
@@ -53,19 +61,25 @@ class Favorites extends React.Component<Props, null> {
           renderTabBar={props => (
             <View>
               <Title>Saves &amp; Follows</Title>
-              <TabBar {...props} />
+              <ScrollableTabBar {...props} />
             </View>
           )}
         >
-          <Tab tabLabel="Works">
+          <ScrollableTab tabLabel="Works">
             <ArtworksRenderer render={renderWithLoadProgress(Artworks)} />
-          </Tab>
-          <Tab tabLabel="Artists">
+          </ScrollableTab>
+          <ScrollableTab tabLabel="Artists">
             <ArtistsRenderer render={renderWithLoadProgress(Artists)} />
-          </Tab>
-          <Tab tabLabel="Categories">
+          </ScrollableTab>
+          <ScrollableTab tabLabel="Categories">
             <CategoriesRenderer render={renderWithLoadProgress(Categories)} />
-          </Tab>
+          </ScrollableTab>
+          <ScrollableTab tabLabel="Fairs">
```

Comment:
You can actually go ahead and remove that tab in a separate commit that we can just revert it once fair following is ready